src/emulator/env.ts

```diff
@@ -85,3 +87,30 @@ export async function getCredentialsEnvironment(
   }
   return credentialEnv;
 }
+
+export function maybeUseMonospacePortForwarding(emulatorInfos: EmulatorInfo[]): EmulatorInfo[] {
+  const portForwardingHost = process.env.MONOSPACE_PORT_FORWARDING_HOST;
+  if (process.env.MONOSPACE_ENV && portForwardingHost) {
```

Comment:
can this variable be FIREBASE_STUDIO_ENV or just not be checked at all?

Reply:
This one already exists and is in use elsewhere (in firebase-vscode). However, it is  somewhat redundant here, so I'll remove it